### ISSUE TITLE
Not finding my shopify shop name, redirects to embeds.shopify.com instead

### ISSUE THREAD

**Issue by websitedeptuk** [secretpizzaparty/shopify-connect-for-woocommerce#24]
On activating your plugin, I found that it redirects to "embeds.myshopify.com", which is an empty sample store, not my Shopify store. 

"Embeds" seems to be the backup you have specified in case the plugin can't find the store that the official  Shopify plugin uses?

I manually changed the 'secp_shop' field in my database, and this seems to have resolved it for me.

**Comment by nickhamze** [comment]
This pesky bug again. I thought we'd finally fixed it. Any ideas @jkudish ?

**Comment by jkudish** [comment]
@websitedepuk where exactly did you see embeds as the url/redirect? 

That's the default that the shopify plugin sets (not in our control) but our plugin should ignore it for redirect purposes. It's possible we missed a spot. 

Let me know!

**Comment by websitedeptuk** [comment]
In shopify-connect.php, on line 113, I can see 'embeds.shopify.com'.

**Comment by jkudish** [comment]
Yep that's where we ignore it...

Can you help me out by letting me know where you saw the redirect please?

**Comment by websitedeptuk** [comment]
The problem occurred when I clicked on Woocommerce in the left menu bar and that took me straight to embeds.shopify.com.

I don't know if this helps, but in terms of plugin installation, initially, I installed and activated the Shopify plugin. Then I deactivated that and installed and activated the Woocommerce plugin.

When I discovered your plugin to connect the two I installed and activated it, but it told me that I needed both Woocommerce and Shopify plugins active. I reactivated the Shopify plugin and then activated your plugin.
